tests/integration/test_starkExchange.py

```diff
@@ -0,0 +1,73 @@
+from typing import Callable
```

Comment:
don't mix casing in file name: use instead `test_starkexchange.py` or `test_stark_exchange.py`

Reply:
Ok!